modules/deploy/pages/deployment-option/cloud/api/cloud-api-overview.adoc

```diff
@@ -5,14 +5,15 @@
 :page-api-slot: overview
 :page-beta: true
 
-The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management and manage Redpanda Cloud resources.
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud, including xref:deploy:deployment-option/cloud/serverless.adoc[Serverless]. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management and manage Redpanda Cloud resources.
```

Comment:
Link to Terraform content, please.

Reply:
@vladoschreiner I believe our Terraform content will not be delivered until September -- @micheleRP could you confirm please?

Reply:
Correct: I was told that the Redpanda Terraform provider for Cloud won't be released until all Cloud clusters have been migrated to v24.2. September is a guess, it may be sooner than that.

Reply:
@vladoschreiner We will create a new/separate ticket to ensure the link to Terraform provider doc gets created once it is released/available.